src/core/config.js

```diff
@@ -47,6 +47,7 @@ const config = {
 		childModules: [],
 		testsRun: 0,
 		unskippedTestsRun: 0,
+		testsNotRun: 0,
```

Comment:
I'm finding this terminology to be quite difficult to parse. As far as I can tell, this is actually representing the tests that _should_ not be ran (not those that happen to have not ran yet). 

Assuming that I've understood the logic here, here are some possible other names:

* `testsToBeIgnored`
* `excludedTests`

Also, why do we need both `testsRun`, `unskippedTestsRun`, and `testsNotRun` (or whatever name we decide to add). Do we actually need to track all three of these separately? It seems like we should only have two concepts:

* the count of tests that we have actually ran (currently `testsRun`)
* the count of tests that we plan to ignore (currently `testsNotRun`, but hopefully we can find a better name)

In other words, skipped tests are just another form of test to be ignored (just the same as tests that are filtered out).

Reply:
Renamed this `testsNotRun ` to `testsIgnored` and removed `unskippedTestsRun`.
Tests that are skipped or filtered out will be accounted for by `testsIgnored`.